src/tlog-test-json-esc.c

```diff
@@ -95,7 +98,7 @@ main(void)
                      _out_ptr, _out_len, _in_str)               \
     TEST_BUF_BUF(_name_token, _exp_res_len,                     \
                  _out_ptr, _out_len, _in_str, strlen(_in_str))
-                 
+
```

Comment:
Please do not remove this whitespace in this commit. Make a separate commit, if you want to clean it up.